core-common/src/main/java/org/apache/kylin/common/util/CliCommandExecutor.java

```diff
@@ -164,7 +164,8 @@ private Pair<Integer, String> runNativeCommand(String command, Logger logAppende
 
             try {
                 int exitCode = proc.waitFor();
-                return Pair.newPair(exitCode, result.toString());
+                String info = exitCode != 0 ? "Job execute failed, see kylin log or spark log for more info." : "Success";
+                return Pair.newPair(exitCode, info);
```

Comment:
This changed the behavior, may cause more issues.

Suggestion: how about adding a max. cap to the "result" buffer, so that only print out the last X characters, say 1K or 10K.

Reply:
Thanks for suggest to this, this is a better way

Reply:
I've revised it.